src/xbacklight.cc

```diff
@@ -0,0 +1,443 @@
+/*
```

Comment:
Added in https://github.com/javihernandez/node-xbacklight/commit/b19273f188df4832e9fd2090c3d7777413592f57